test/cri-containerd/createcontainer_test.go

```diff
@@ -1191,3 +1192,92 @@ func Test_CreateContainer_Mount_NamedPipe_WCOW(t *testing.T) {
 	}
 	runCreateContainerTest(t, wcowHypervisorRuntimeHandler, request)
 }
+
+func Test_CreateContainer_GMSA_WCOW(t *testing.T) {
```

Comment:
What is this testing that we don't get with the `RunContainer` test?

Reply:
This was just the first test I had made to validate that nothing failed to parse/returned an error through the process, it doesn't afford us anything that the USERDNSDOMAIN one doesn't.